src/components-examples/material/table/table-reorderable/table-reorderable-example.ts

```diff
@@ -0,0 +1,46 @@
+import { Component } from '@angular/core';
+import { CdkDragDrop, moveItemInArray } from '@angular/cdk/drag-drop';
+
+/**
+ * @title Table with re-orderable columns
+ */
+@Component({
+  selector: 'table-reorderable-example',
+  templateUrl: './table-reorderable-example.html',
+  styleUrls: [ './table-reorderable-example.css' ]
+})
+export class TableReorderableExample {
+  private originalColumnPositions: string[] = ['position', 'name', 'weight', 'symbol'];
+  columns: string[] = Object.assign([], this.originalColumnPositions);
```

Comment:
I don't `Object.assign` is appropriate here. If you want to clone the array you can do `this.originalColumnPositions.slice()`.

Reply:
Done. TIL